miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PrinterRestController.java

```diff
@@ -246,6 +246,32 @@ public long boxPositions(@PathVariable("printerId") Long printerId, @RequestBody
             .map(WhineyFunction.rethrow(this::loadBarcodable)));
   }
 
+  @PostMapping(value = "{printerId}/boxpositionsbycolumn", headers = {"Content-type=application/json"})
+  @ResponseBody
+  public long boxPositionsByColumn(@PathVariable("printerId") Long printerId,
+      @RequestBody BoxPositionPrintRequest request)
```

Comment:
We don't need a new endpoint for this. Make sort order a field in `BoxPositionPrintRequest` and handle that in the existing endpoint.